### PR TITLE
Journal list integration

### PR DESCRIPTION
**Pull request by DanaeBouwer**
**Integration of JSTOR journal list and fixing errors**

_Summary_
Integrated the journal_title_scraper into the masterlist_scraper to automatically pull the journals to scrape from JSTOR. Also fixed 2 errors encountered:
- element has 0 size
- connection interrupted

### CODE REVIEW

File: masterlist_scraper.py [hunk 1]
```diff
@@ -88,29 +107,54 @@
...
+            
+        except Exception as e:
+            print(e)
+            input()
 
         old_name=os.path.join(directory,'citations.txt')
         new_name=os.path.join(directory,issue_url.split("/")[-1]+'.txt')
+
+        # Wait for download to complete
+        while not os.path.isfile(old_name):
```
Comment (wenkanglu) [inline masterlist_scraper.py]: This loop is potentially infinite if the file is never successfully downloaded. Consider adding a maximum_duration constant that signifies the longest time we are willing to wait for a download before failing.

**Review by wenkanglu — CHANGES_REQUESTED** [review]
Looks good overall. Just a minor comment regarding a while loop. I would make the suggested change and update this pull request as practice.

**Review by wenkanglu — APPROVED** [review]
Nice, easy fix.

**Comment by wenkanglu** [comment]
To whoever is merging (Takunda/Danae), please select the "Squash and Merge" option when doing so.

**Review by takundachirema — APPROVED** [review]

**Pull request merged**